ADAL/src/ui/mac/ADCredentialCollectionController.m

```diff
@@ -36,11 +61,38 @@ -(id) init
     self = [super init];
     if(self)
     {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-        [NSBundle loadNibNamed:@"ADCredentialViewController" owner:self];
-#pragma clang diagnostic pop
-//        loaded = [[NSBundle mainBundle] loadNibNamed:@"ADCredentialViewController" owner:self topLevelObjects:nil];
+        //Generate the NTLM input dialog by code for Mac
+        //usename field
+        _usernameLabel = [[NSTextField alloc] initWithFrame:NSMakeRect(USERNAME_LABEL_X, USERNAME_LABEL_Y, USERNAME_LABEL_WIDTH, USERNAME_LABEL_HEIGHT)];
+        SAFE_ARC_AUTORELEASE(_usernameLabel);
+        [_usernameLabel setStringValue:@"User Name"];
```

Comment:
Look up the NSLocalizedString APIs. Seeing a hardcoded string like this makes loc people nervous.